azure/table_azure_compute_virtual_machine_scale_set_network_interface.go

```diff
@@ -0,0 +1,259 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-05-01/network"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureComputeVirtualMachineScaleSetNetworkInterface(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_compute_virtual_machine_scale_set_network_interface",
+		Description: "Azure Compute Virtual Machine Scale Set Network Interface",
+		List: &plugin.ListConfig{
+			ParentHydrate: listAzureComputeVirtualMachineScaleSets,
+			Hydrate:       listAzureComputeVirtualMachineScaleSetInterfaces,
+		},
+		Columns: azureColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "Name of the scale set network interface.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "scale_set_name",
+				Description: "Name of the scale set.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID").Transform(extractScaleSetFromID),
+			},
+			{
+				Name:        "id",
+				Description: "The unique id identifying the resource in subscription.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromGo(),
+			},
+			{
+				Name:        "provisioning_state",
+				Description: "he provisioning state of the network interface resource. Possible values include: 'Succeeded', 'Updating', 'Deleting', 'Failed'.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(extractScaleSetNetworkInterfaccePrpperties, "ProvisioningState"),
+			},
+			{
+				Name:        "type",
+				Description: "The type of the resource in Azure.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "mac_address",
+				Description: "The MAC address of the network interface.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Interface.MacAddress"),
+			},
+			{
+				Name:        "enable_accelerated_networking",
+				Description: "If the network interface is accelerated networking enabled.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromP(extractScaleSetNetworkInterfaccePrpperties, "EnableAcceleratedNetworking"),
```

Comment:
```suggestion
				Transform:   transform.FromP(extractScaleSetNetworkInterfaceProperties, "EnableAcceleratedNetworking"),
```
Please make the changes across all such occurrences.